<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1121</width>
      <height>701</height>
     </rect>
    </property>
    <widget class="QWidget" name="start_page">
     <widget class="QPushButton" name="imageButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>60</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>StegoImage</string>
      </property>
     </widget>
     <widget class="QPushButton" name="audioButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>120</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>StegoAudio</string>
      </property>
     </widget>
     <widget class="QPushButton" name="videoButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>180</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>StegoVideo</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="imagePage">
     <widget class="QPushButton" name="imageExtractNavBtn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>280</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Extract Message</string>
      </property>
     </widget>
     <widget class="QLabel" name="imagePageTitle">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Image Steganography</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="imageHideNavBtn">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>280</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Hide Message</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="imageHidePage">
     <widget class="QGroupBox" name="imageStegoGroup">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>471</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>Raw Image</string>
      </property>
      <widget class="QLabel" name="imageStegoPicLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>451</width>
         <height>401</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="imageHideTitle">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Image Hide</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="imageRawGroup">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>70</y>
        <width>471</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>Stego Image</string>
      </property>
      <widget class="QLabel" name="imageRawPicLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>451</width>
         <height>401</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="imageMenuGroup">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>510</y>
        <width>1001</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Menu</string>
      </property>
      <widget class="QPushButton" name="backImagePageNavBtn">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>70</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Back to Image Stegano Menu</string>
       </property>
      </widget>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>491</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="imageLeftForm">
        <item row="0" column="0">
         <widget class="QLabel" name="imgRawPathLbl">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Raw Image Path</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="imgRawPathGrid">
          <item row="0" column="1">
           <widget class="QToolButton" name="imgRawPathBtn">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="imgRawPathInp">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="imgFileInpPathLbl">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>File Input Path</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="imgMethUsedLbl">
          <property name="text">
           <string>Method Used</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="imgMethUsedInp"/>
        </item>
        <item row="1" column="1">
         <layout class="QGridLayout" name="imgFileInpPathGrid">
          <item row="0" column="1">
           <widget class="QToolButton" name="imgFileInpPathBtn">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="imgFileInpPathInp">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="imgHideFileBtn">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>20</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Hide File</string>
       </property>
      </widget>
      <widget class="QWidget" name="formLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>20</y>
         <width>291</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="imageRightForm">
        <item row="0" column="0">
         <widget class="QLabel" name="imgKeyLbl">
          <property name="text">
           <string>Key (If Needed)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="imgKeyInp"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="imgEncLbl">
          <property name="text">
           <string>Encryption</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="imgEncInp">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="imgNonSeqLbl">
          <property name="text">
           <string>Non Sequential</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="imgNonSeqInp">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="imgSaveOutBtn">
       <property name="geometry">
        <rect>
         <x>910</x>
         <y>20</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Output</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="imageExtractPage">
     <widget class="QGroupBox" name="imageInpGroupE">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>70</y>
        <width>471</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>Stego Image Input</string>
      </property>
      <widget class="QLabel" name="imageInpPicLabelE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>451</width>
         <height>401</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="imageExtractTitle">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Image Extract</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="imageMenuGroupE">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>510</y>
        <width>1001</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Menu</string>
      </property>
      <widget class="QPushButton" name="backImagePageNavBtnE">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>70</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Back to Image Stegano Menu</string>
       </property>
      </widget>
      <widget class="QWidget" name="formLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>491</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="imageLeftFormE">
        <item row="0" column="0">
         <widget class="QLabel" name="imgStegPathLblE">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Steg Image Path</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="imgStegPathGridE">
          <item row="0" column="1">
           <widget class="QToolButton" name="imgStegPathBtnE">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="imgStegPathInpE">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="imgFileOutPathLblE">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>File Output Path</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="imgMethUsedLblE">
          <property name="text">
           <string>Method Used</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="imgMethUsedInpE"/>
        </item>
        <item row="1" column="1">
         <layout class="QGridLayout" name="imgFileOutPathGridE">
          <item row="0" column="1">
           <widget class="QToolButton" name="imgFileOutPathBtnE">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="imgFileOutPathInpE">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="imgExtractButton">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>20</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Extract</string>
       </property>
      </widget>
      <widget class="QWidget" name="formLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>20</y>
         <width>291</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="imageRightFormE">
        <item row="0" column="0">
         <widget class="QLabel" name="imgKeyLblE">
          <property name="text">
           <string>Key (If Needed)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="imgKeyInpE"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="imgEncLblE">
          <property name="text">
           <string>Encryption</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="imgEncInpE">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="imgSaveOutButtonE">
       <property name="geometry">
        <rect>
         <x>910</x>
         <y>20</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Output</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="audioPage">
     <widget class="QPushButton" name="AHidePageButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>100</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Hide</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AExtractPageButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>170</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Extract</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="audioHidePage">
     <widget class="QLabel" name="fileLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>File yang ingin disembunyikan:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="audioTextEdit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>90</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="fileTextEdit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>120</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="keyLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Kunci Vigenere:</string>
      </property>
     </widget>
     <widget class="QLabel" name="audioLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>File audio:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="keyTextEdit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>150</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="AEnkripsiCheck">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>220</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Enkripsi File</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ASeqCheck">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>250</y>
        <width>79</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Sekuensial</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AHideButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>220</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>HIde</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="videoPage"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
